### ISSUE TITLE
Swift 4.2 macOS CommonCrypto conflict

### ISSUE THREAD

**Issue by quanvo87** [IBM-Swift/BlueCryptor#41]
Latest macOS SDK includes a module map for CommonCrypto.

So IBM's is no longer necessary, and produces a conflict and compile failure.

**Comment by ianpartridge** [comment]
Yes, we added https://github.com/IBM-Swift/BlueCryptor/blob/master/Package%40Swift-4.swift which doesn't include the IBM CommonCrypto repo.

**Comment by ianpartridge** [comment]
The 4.0 and 4.1 Package.swifts do.
